Makefile

```diff
@@ -21,6 +21,7 @@ include $(ROOT_DIR_RELATIVE)/common.mk
 
 # Active module mode, as we use go modules to manage dependencies
 export GO111MODULE=on
+unexport GOPATH
```

Comment:
I though it doesn't make sense to support various variants of GOPATH setups when we can just "disable" GOPATH. As far as I can see it works.